app/views/Settings/GoogleMapsImport.js

```diff
@@ -14,30 +14,35 @@ export const GoogleMapsImport = ({ navigation }) => {
     navigation.navigate('ImportScreen');
   };
 
-  return (
-    <>
-      <TitleRow>
-        <SvgXml xml={Icons.GoogleMapsLogo} />
-        <Title use='body1'>{t('import.google.title')}</Title>
-      </TitleRow>
-
-      <ParagraphContainer>
-        <Typography use='body2'>{t('import.subtitle')}</Typography>
-      </ParagraphContainer>
-
-      <Button
-        secondary
-        label={t('import.button_text')}
-        onPress={importPressed}
-      />
-
-      <ParagraphContainer>
-        <Typography use='body3' secondary monospace>
-          {t('import.google.disclaimer')}
-        </Typography>
-      </ParagraphContainer>
-    </>
-  );
+  // GoogleMapsImport is only enabled on the GPS app
+  if (config.tracingStrategy === 'gps') {
+    return (
```

Comment:
Also what do you think about pulling the conditional up altogether?  That is the settings page is the component that determines if the GoogleImports should be rendered at all.

Reply:
Yeah, I was curious to hear other's thoughts on this. Agreed on the ugly conditional render - my thought with pulling it into the component rather than the parent Settings page was that regardless of where the `<GoogleMapsImport />` is, it should never render is it isn't the GPS build.

The big tradeoff there is that it's way less apparent in the parent component. 

I think I prefer your snippet - same thing goes for the new `<LocationPermission />` component being introduced here, so I'll make the same update there.

Reply:
I also prefer: `{isGPS && <GoogleMapsImport />}`

and +100 to pulling up:

It makes more sense to me that `Settings.js` owns the switch logic, not this component:

- It's odd for a component to conditionally switch itself entirely to null.
- Settings has far more reason to know about gps/bte context.

Reply:
Cool - thanks for the feedback 👍 I have implemented this.